docs: @fuel-wallet/react (#884)
FuelLabs/fuels-wallet: packages/docs/docs/dev/accounts.mdx

  commentBlock="eventCurrentAccount"
/>
<Examples.Events.CurrentAccount />

### With React

> You can keep track of the current account being used just by invoking the `useAccount` hook as shown below:

Reviewer: ```suggestion
You can keep track of the current account when using React. Using the `useAccount` hook:
```

  commentBlock="eventCurrentAccount"
/>
<Examples.Events.CurrentAccount />

## With React

> You can keep track of the current account when using React. Using the `useAccount` hook as shown below:

Reviewer 2: ```suggestion
You can keep track of the current account when using React using the `useAccount` hook as shown below:
```
